src/components/Location/LocationTree.tsx

```diff
@@ -16,13 +17,15 @@ interface LocationNodeProps {
   location: LocationList;
   isLast: boolean;
   datetime?: string;
+  endDatetime?: string;
```

Comment:
props could be confusing, update datetime to be more specific

Reply:
☝️  Rithvik's comment hasn't been addressed. At first glance, name doesn't tell me the difference between datetime and endDatetime.

Make sure to change elsewhere as well.